CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-# dbt_iterable v0.RELEASE.RELEASE
+# dbt_iterable v0.11.1
```

Comment:
There are changes to when `experiment_id` is or is not being brought in this model, plus the new aggregate metrics, which I believe classifies as a schema change. Should we consider making this a breaking change and bumping the version up?

Reply:
good point-- bumped to breaking change